demo/demoapp/urls.py

```diff
@@ -2,12 +2,14 @@
 
 from demoapp.cruds import Personclass, Countryclass, MenuItemclass
 from demoapp.views import create_notification_view
+from .views import formView
 
 pclss = Personclass()
 countryclss = Countryclass()
 menuclss = MenuItemclass()
 
 urlpatterns = [
-        path('create/notification', create_notification_view)
+        path('create/notification', create_notification_view),
+        path('form', formView),
```

Comment:
Path needs to be significant, so path('form', ...) is a bad name for this view

Reply:
Changed it to knobwidget/testform